docs/ssl_errors.md

```diff
@@ -0,0 +1,65 @@
+## Troubleshooting SSL errors
+
+There are several issues that may arise related to SSL certificates.
+Check [badssl site](https://badssl.com/) for a comprehensive list.
+It is very common, though, environments where `self-signed` certificates are deployed.
+
+It can be `SSL certificate problem: self signed certificate`
+
+```bash
+# curl https://self-signed.badssl.com/
+curl: (60) SSL certificate problem: self signed certificate
+```
+
+Or it can be `self signed certificate in certificate chain`
+
+```bash
+$ curl https://3scale-admin.5e1f.apps.rhpds.openshift.opentlc.com
+curl: (60) SSL certificate problem: self signed certificate in certificate chain
+```
+
+The *recommended* way to solve this issues is fetching remote host certificate
+and using that certificate with the toolbox.
+
+You can get the remote host certificate by using OpenSSL:
+
+```bash
+echo | openssl s_client -showcerts -servername self-signed.badssl.com -connect self-signed.badssl.com:443 2>/dev/null | openssl x509 > self-signed-cert.pem
+```
+
+Then tell the toolbox to use that certificate using `SSL_CERT_FILE` environment variable.
+
+```bash
+SSL_CERT_FILE=self-signed-cert.pem 3scale command params
```

Comment:
I think it is worth to mention that you can use SSL_CERT_FILE with curl too, so people can verify it works there.